internal/schema/extractor/schema_test.go

```diff
@@ -544,6 +544,8 @@ level: string | enum=debug,info,warn | default=info
 status: 'string | enum="active","inactive","pending" | default="active"'
 priority: 'integer | enum=1,2,3 | default=1'
 code: 'integer | enum="1","2","3" | default="1"'
+nodeVersion: 'string | default="18" enum="16,18,20,22"'
```

Comment:
Do we really need to support this? It conflicts with [1] when handling comma-separated values, and some might interpret it as a single item rather than multiple values.
I think we should drop this along with code: 'integer | enum="1","2","3" | default="1"'. While initially added for user convenience, these variations create more confusion than they're worth.

[1] https://github.com/openchoreo/openchoreo/blob/main/docs/templating/schema.md#quoting-and-escaping